src/main/java/org/codelibs/fess/query/TermQueryCommand.java

```diff
@@ -164,7 +164,12 @@ protected QueryBuilder convertDefaultTermQuery(final FessConfig fessConfig, fina
         final BoolQueryBuilder boolQuery =
                 buildDefaultQueryBuilder(fessConfig, context, (f, b) -> buildMatchPhraseQuery(f, text).boost(b * boost));
         final Integer fuzzyMinLength = fessConfig.getQueryBoostFuzzyMinLengthAsInteger();
-        if (fuzzyMinLength >= 0 && text.length() >= fuzzyMinLength) {
+
+        // Do not add fuzzy queries if the single-word query is surrounded by double quotes
+        final String exactWordMatchLookupRegex = ".*\"" + text + "\".*";
```

Comment:
I want to avoid using this regexp in the term query component.

Reply:
By "avoid using this regexp", do you mean that only this regexp `".*\"" + text + "\".*"` is not OK?
Or should I find another approach (that doesn't use regexp) instead?

Sorry for not understanding it correctly.

Reply:
We need another approach.

Reply:
I understand the situation. I'll extend Lucene's QueryParser to handle a single-phrase query as a phrase query.